### DISCUSSION TITLE
Why HTTP API mode removes the scheduler?

### DISCUSSION THREAD

**Discussion by kopiro** [containrrr/watchtower#938] [Feature Requests] [answered]
From the docs, *Runs Watchtower in HTTP API mode, only allowing image updates to be triggered by an HTTP request. For details see HTTP API.*

The question here is why can't I have both? I'm exactly in the scenario where I need a scheduler (maybe every day, or every hour) but also support to update on the spot via HTTP.

For now I've achieved the same using an external cron the calls the /v1/update, but I would like to have the internal scheduler instead.

**Comment by simonaronsson** [comment, marked as answer]
Hi @kopiro,

I understand your use case and it makes total sense. We're tracking that feature in issue https://github.com/containrrr/watchtower/issues/897.

Best,
Simme
